scripts/build.ts

```diff
@@ -0,0 +1,3 @@
+import { exec } from 'shelljs';
+
+exec('rm -rf lib && tsc -b')
```

Comment:
Just tried running `tsc -b` on the master branch, and discovered that there is a few issues more then the one for `language.service.ts`. Do you mind fixing these? it should be pretty straight forward

Reply:
@arelstone Hi there, I don't think i will have time to contribute further to this P.R in the short term as Im not sure if we are going ahead with Tradoura platform. 

We are looking for a solution that allows better integration with Git to store translations, rather than a database, so until we resolve that - Ill wont have time to contribute further. Apologies.